doc/perf.md

```diff
@@ -236,22 +228,16 @@ Segmentation fault
 $
 ```
 
-But with optimizations enabled, the program does not crash even if I type `y`:
+But with optimizations enabled, the program does not crash even when typing `y`:
 
 ```
 $ ./jou -O3 asd.jou
 Crash this program? (y/n) y
 $
 ```
 
-The optimizer assumes that you don't attempt to access the value of a `NULL` pointer.
-In other words, it thinks that the `x = *foo` code will never run,
-and it can therefore be ignored.
-
-Correctly written Jou code does not break when optimizations are enabled.
-For example, accessing a `NULL` pointer simply isn't a good way to exit a program;
-your program shouldn't ever do it.
-If you want the program to crash, you can use the `abort()` function, for example:
+The optimizations make the program ignore the code to access the value of a `NULL` pointer.
+Essentially it thinks that the `x = *foo` code will never run, because you aren't supposed to access the value of a NULL pointer. This code will thus get ignored. NOTE: if y0ou want the program to crash with optimizations on, you should do so using `abort()` function for example:
```

Comment:
```suggestion
Essentially it thinks that the `x = *foo` code will never run, because you aren't supposed to access the value of a NULL pointer. This code will thus get ignored.

A better way to crash the program would be using the `abort()` function. For example:
```

Reply:
I would keep the NOTE: part in. This is distraction - the text isn't about the ways to crash the program, but about the jou optimizations.

Reply:
Maybe like this?

```suggestion
Essentially it thinks that the `x = *foo` code will never run, because you aren't supposed to access the value of a NULL pointer. This code will thus get ignored.

If you actually want to crash the program, don't use a NULL pointer to do it. Use the `abort()` function instead:
```

Reply:
I'd still keep the NOTE in (or use some other indicator that the following is not something to focus on). The focus for a reader goes a long way, I've discovered it the painful way myself :)

Reply:
How about "As a side note, if you actually want to crash ..."?

Reply:
Yeah, how about just 'Note:' ?

Reply:
Sure, that is fine :)